### PR TITLE
Removed case-sensitivity

### PR DESCRIPTION
**Pull request by DavidWChen**
## Description
Saw this issue in the data base (emails are case sensitive); added a lower() to prevent it from happening and also put it in email to keep it uniform.
screen shot 2018-01-24 at 8 42 46 pm

Also removed regex allowing emails without (g.)

## Pls do 
- [x] Tested and working
- [ ] Unit tests

## Test plan
Postman
screen shot 2018-01-24 at 10 41 47 pm
screen shot 2018-01-24 at 10 42 05 pm
screen shot 2018-01-24 at 10 42 18 pm

### CODE REVIEW

**Comment by marktai** [comment]
please rebase your branch

File: src/users/views.py [hunk 1]
```diff
@@ -80,18 +80,18 @@ class CreateUser(generics.CreateAPIView):
     
     @transaction.atomic
     def post(self, request):
-        if User.objects.filter(email=request.data['email']).exists():
+        if User.objects.filter(email=request.data['email'].lower()).exists():
```
Comment (marktai) [inline src/users/views.py]: use this: 
https://stackoverflow.com/questions/11743207/django-model-case-insensitive-query-filtering

Comment (marktai) [reply src/users/views.py]: also, i'd like to see a test case to ensure this behavior exists after people edit code

**Comment by DavidWChen** [comment]
If my commits now look like this, it means I rebased correctly, right?

**Comment by marktai** [comment]
ya looks good.

**Pull request merged**